src/components/SearchResultsPage/SearchResultsPage.js

```diff
@@ -13,22 +14,31 @@ const getParsedQuery = (location) => {
     return parsedQuery.kw
 }
 
+const getPage = (location) => {
+    const page = queryString.parse(location.search);
+    return page.p
+}
+
 const SearchResultPage = (props) => {
+    
     const [coursesState, setCoursesState] = useState({
         coursesObj: null,
         isLoading: false,
+        page: 1,
         error: null
     });
 
 
     const [searchInputState, setSearchInputState] = useState({
         input: ''
     });
-
+    
     // const jsonObj = dummyJSON;
     let location = useLocation();
-    const api_url = 'http://localhost:8080/es_api/?keyword='
+    const api_url = 'http://localhost:8080/es-api/?keyword='
```

Comment:
This endpoint specification should be moved into configuration

Reply:
Mac will handle this in his commits